docs/Video Streaming for Navigation Apps/Introduction/index.md

```diff
@@ -29,10 +45,47 @@ lifecycleConfig.appType = .navigation
 let streamingConfig = SDLStreamingMediaConfiguration(securityManagers: [OEMSecurityManager.self])
 let config = SDLConfiguration(lifecycle: lifecycleConfig, lockScreen: .enabled(), logging: .default(), streamingMedia: streamingConfig, fileManager: .default())
 ```
+!@
+
+@![android]
+```java
+SdlManager.Builder builder = new SdlManager.Builder(this, APP_ID, APP_NAME, listener);
+
+Vector<AppHMIType> hmiTypes = new Vector<AppHMIType>();
+hmiTypes.add(AppHMIType.NAVIGATION);
+builder.setAppTypes(hmiTypes);
+
+List<? extends SdlSecurityBase> securityManagers = new ArrayList();
+securityManagers.add(OEMSecurityManager1.class);
+securityManagers.add(OEMSecurityManager1.class);
+builder.setSdlSecurity(securityManagers);
+
+MultiplexTransportConfig mtc = new MultiplexTransportConfig(this, APP_ID, MultiplexTransportConfig.FLAG_MULTI_SECURITY_OFF);
+mtc.setRequiresHighBandwidth(true);
+builder.setTransportType(transport);
+
+sdlManager = builder.build();
+sdlManager.start();
+```
+!@
 
+@![iOS,android]
 !!! IMPORTANT
 When compiling, you must make sure to include all possible OEM security managers that you wish to support.
 !!!
+!@
 
+@![iOS]
 ## Keyboard Input
-To present a keyboard (such as for searching for navigation destinations), you should use the `SDLScreenManager`'s keyboard presentation feature. For more information, see the [Popup Menus and Keyboards](Displaying a User Interface/Popup Menus and Keyboards) guide.
\ No newline at end of file
+To present a keyboard (such as for searching for navigation destinations), you should use the `SDLScreenManager`'s keyboard presentation feature. For more information, see the [Popup Menus and Keyboards](Displaying a User Interface/Popup Menus and Keyboards) guide.
+!@
+
+@![android]
```

Comment:
Android and iOS should have a section for presenting menus and such using other templates.